app/models/ead_processor.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+class EadProcessor
+  require 'nokogiri'
+  require 'zip'
+  require 'fileutils'
```

Comment:
Does the code still work the same w/o these requires? Because nokogiri, zip and arc light are all in the bundle, I think they are likely already required. I also think fileutils is likely required before you get to this model as well?

Reply:
It works without them. Thank you for catching that.